### DISCUSSION TITLE
Support for Windows

### DISCUSSION THREAD

**Discussion by leshchenko** [firebase/flutterfire#6071] [Q&A] [answered]
Hi, I'm currently working on adding the windows platform in my current flutter project, and **cloud_firestore** - is a core lib that is used in the app.
Are there any plans to add an implementation for the Windows platform?

**Comment by markbreuss** [comment]
We have the same problem.
Is there a rough timeline for adding Windows support?

**Reply by haspdev** [reply]
We have the same problem I am waiting for windows support. 
 I need the firebase database, firebase auth, firestore, firebase storage.  please add support for windows ASAP. we losing customers, due to the absence of windows support.

**Comment by worldwidee** [comment]
same problem

**Comment by barbalex** [comment, marked as answer]
Duplicate of https://github.com/FirebaseExtended/flutterfire/discussions/5557
